paper-badge.html

```diff
@@ -181,7 +182,7 @@
 
       _updateTarget: function() {
         this._target = this.target;
-        this.async(this.notifyResize, 1);
+        requestAnimationFrame(this.notifyResize.bind(this));
```

Comment:
No, I meant: should we create a `_boundNotifyResize` and bind it once (like https://github.com/PolymerElements/iron-behaviors/blob/master/iron-control-state.html#L49), rather than binding the function _every time_ the target changes